src/Uno.UI/UI/Xaml/Media/Animation/FadeInThemeAnimation.cs

```diff
@@ -26,7 +26,9 @@ public FadeInThemeAnimation()
 		}
 
 		public static DependencyProperty TargetNameProperty { get; } = DependencyProperty.Register(
-			"TargetName", typeof(string), typeof(FadeInThemeAnimation), new FrameworkPropertyMetadata(null));
+#pragma warning disable Uno0002_Internal // String dependency properties (in *most* cases) shouldn't have null default value.
+			"TargetName", typeof(string), typeof(FadeInThemeAnimation), new FrameworkPropertyMetadata(null)); // TODO: What the default value should be? null or string.Empty?
```

Comment:
It looks like on Windows the value is string.Empty. What makes you think otherwise? (same for the cases other files)

Reply:
I tried to validate it now and indeed it's string.Empty. The property that I wasn't able to verify on UWP is `Storyboard.TargetNameProperty`

![image](https://user-images.githubusercontent.com/31348972/133666163-5d78789c-21cb-4bf1-9106-4531dc98a050.png)

But I see now that I should have used `new ObjectAnimationUsingKeyFrames().GetValue(Storyboard.TargetNameProperty)` instead to validate. I'll fix them.